packages/kbn-relocate/README.md

```diff
@@ -10,16 +10,16 @@ You must have `gh` CLI tool installed. You can install it by running:
 brew install gh
 ```
 
-You must also configure your "default" kibana repo in `gh`, so that it can find PRs.:
-
-```sh
-gh repo set-default elastic/kibana
-```
-
 You must have `elastic/kibana` remote configured under the name `upstream`.
 
 You must have a remote named `origin` pointing to your fork of the Kibana repo.
 
+It's highly recommended that you disable the pre-commit hook, as it often causes the script to fail:
+
+```sh
+mv .git/hooks/pre-commit .git/hooks/pre-commit.sample
+```
```

Comment:
instead of doing this, the script could add `--no-verify` to the `commit` command.